osgi.specs/docbook/core/003/framework.module.xml

```diff
@@ -3110,10 +3158,22 @@ B: Bundle-SymbolicName: B
         </listitem>
       </itemizedlist>
 
-      <para>The above only applies to the import and export package
+      <para>The above only applies to the <code>Import-Package</code> and <code>Export-Package</code>
       definitions of a bundle. For namespaces other than
       <code>osgi.wiring.package</code>, a requirement definition of a bundle
       may be wired to a capability definition of that same bundle.</para>
+
+      <para>
+	  	<emphasis role="bold">Notes on <emphasis>External</emphasis></emphasis>: 
+      If the resolver choses to resolve to an export statement in another bundle 
+		  the framework must <emphasis>discard</emphasis> the export package
+		  of the current bundle. This can lead to cases where consumers 
+		  for a version of the current bundle become <emphasis>unresolvable</emphasis>. 
+		  This effect can happen if the import has <emphasis>no</emphasis> version range 
```

Comment:
Actually this can happen always, it only depends on the circumstances and used import/export ranges.

Reply:
I removed the last sentence.

> This effect can happen if the import has <emphasis>no</emphasis> version range...

Better?